src/components/Link/index.tsx

```diff
@@ -5,8 +5,10 @@ import { withProps } from 'recompose';
 import styled from 'styled-components';
 import asRendition from '../../asRendition';
 import { DefaultProps, RenditionSystemProps } from '../../common-types';
-import { darken, monospace } from '../../utils';
+import { darken, monospace, isExternalUrl, toRelativeUrl } from '../../utils';
+import path from 'path';
 import { align, bold, caps } from '../Txt';
+import { withRouter } from 'react-router-dom';
```

Comment:
we can use the link component directly so we don't have to handle the `ctrl` issue